--- z-push-2.3.9/tools/migrate-2.0.x-2.1.0.php	2018-02-07 12:52:59.000000000 -0500
+++ z-push-2.3.9/tools/migrate-2.0.x-2.1.0.php.NEW	2018-02-24 09:28:49.750315284 -0500
@@ -26,7 +26,7 @@
 ************************************************/
 
 // Please adjust to match your z-push installation directory, usually /usr/share/z-push
-define('ZPUSH_BASE_PATH', "../src");
+define('ZPUSH_BASE_PATH', "/usr/share/zarafa-z-push");
 
 
 
