diff -rup z-push-2.3.6/config.php z-push-2.3.6-config/config.php
--- z-push-2.3.6/config.php	2017-04-26 10:55:28.000000000 -0400
+++ z-push-2.3.6-config/config.php	2017-06-12 20:59:03.473692489 -0400
@@ -27,7 +27,7 @@
  *  Default settings
  */
     // Defines the default time zone, change e.g. to "Europe/London" if necessary
-    define('TIMEZONE', '');
+    define('TIMEZONE', date_default_timezone_get() );
 
     // Defines the base path on the server
     define('BASE_PATH', dirname($_SERVER['SCRIPT_FILENAME']). '/');
@@ -61,7 +61,7 @@
  *           State migration script is available, more informations: https://wiki.z-hub.io/x/xIAa
  */
     define('STATE_MACHINE', 'FILE');
-    define('STATE_DIR', '/var/lib/z-push/');
+    define('STATE_DIR', '/var/lib/zarafa-z-push/state/');
 
 /**********************************************************************************
  *  IPC - InterProcessCommunication
