--- z-push-2.3.9/config.php	2018-02-07 12:52:59.000000000 -0500
+++ z-push-2.3.9/config.php.NEW	2018-02-24 09:32:02.767162259 -0500
@@ -27,7 +27,7 @@
  *  Default settings
  */
     // Defines the default time zone, change e.g. to "Europe/London" if necessary
-    define('TIMEZONE', '');
+    define('TIMEZONE', date_default_timezone_get());
 
     // Defines the base path on the server
     define('BASE_PATH', dirname($_SERVER['SCRIPT_FILENAME']). '/');
@@ -61,7 +61,7 @@
  *           State migration script is available, more informations: https://wiki.z-hub.io/x/xIAa
  */
     define('STATE_MACHINE', 'FILE');
-    define('STATE_DIR', '/var/lib/z-push/');
+    define('STATE_DIR', '/var/lib/zarafa-z-push/state/');
 
 /**********************************************************************************
  *  IPC - InterProcessCommunication
